release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -231,6 +231,18 @@ For more information, see xref:../machine_management/control_plane_machine_manag
 [id="ocp-4-13-nodes"]
 === Nodes
 
+[id="ocp-4-13-pdb-eviction-policy"]
+==== Pod disruption budget (PDB) unhealthy pod eviction policy (Technology Preview)
+
+With this release, specifying an unhealthy pod eviction policy for pod disruption budgets (PDBs) is available as a Technology Preview feature. To use this Technology Preview feature, you must enable the `TechPreviewNoUpgrade` feature set.
+
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. You should not enable this feature set on production clusters.
+====
+
+For more information, see xref:../nodes/pods/nodes-pods-configuring.adoc#pod-disruption-eviction-policy_nodes-pods-configuring[Specifying the eviction policy for unhealthy pods].
```

Comment:
Question: would the link home in on relevant section a little closer if it were: https://57512--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-configuring.html#nodes-pods-configuring-pod-distruption-about_nodes-pods-configuring  ?

Reply:
(I used the docs preview instead of docs.openshift.com)

Reply:
Hey @GroceryBoyJr, I added this note to the additional info section in the first comment, but I could've also mentioned it inline:

> The link at the end of the section won't jump to the right module yet, because it depends on https://github.com/openshift/openshift-docs/pull/57505 being merged first.

So when that PR is merged, this section will jump down to the new section being added there. (The preview for that new section from that PR is this: https://57505--docspreview.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-configuring.html#pod-disruption-eviction-policy_nodes-pods-configuring)